.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: Build and Test
+on: [push, pull_request]
+jobs:
+  # XXX: macOS, Windows
+  ubuntu:
+    strategy:
+      matrix:
+          version: ['7.3', '7.4', '8.0', '8.1']
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install DB2 library
+        # XXX: Can this be optimized to not constantly redownload? Or be put in a better place?
```

Comment:
https://github.com/actions/cache

Reply:
Definitely would be helpful to cache, I think. I know the PRs for python-ibmdb were failing for a while due to flakiness in the IBM download site.